content/funding-your-training.md

```diff
@@ -70,6 +70,19 @@
     - Guernsey
     - Jersey
     - Isle of Man
+    - Undergraduate
+    - Troops to Teachers
+    - Army
+    - Air Force
+    - Navy
```

Comment:
Do we have any content specific to the forces on this page? Having it show up in search with no directly-related content might be confusing.

Reply:
We do, albeit indirectly 

There are Troops to Teachers tax-free bursaries of £40,000 for former military personel undertaking undergraduate degrees leading to QTS in England. You may be eligible for the bursary if you train to teach secondary biology, physics, chemistry, computing, maths or modern foreign languages.